### PR TITLE
updated changelog for AMP 0.4.0

### PR DESCRIPTION
**Pull request by ndegory**
Changelog update (ndegory PRs)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,47 @@
...
+
+* Grafana 4.0 [#518](https://github.com/appcelerator/amp/pull/518)
+* Influxdata stack version 1.1 [#443](https://github.com/appcelerator/amp/pull/443), [#474](https://github.com/appcelerator/amp/pull/474)
+* Robustness [#442](https://github.com/appcelerator/amp/pull/442), [#443](https://github.com/appcelerator/amp/pull/443), [#476](https://github.com/appcelerator/amp/pull/476)
+* grpc code consistency [#586](https://github.com/appcelerator/amp/pull/586) [#585](https://github.com/appcelerator/amp/pull/585) [#584](https://github.com/appcelerator/amp/pull/584) [#583](https://github.com/appcelerator/amp/pull/583)
+* Fix amp stats panic [#533](https://github.com/appcelerator/amp/pull/533) [#534](https://github.com/appcelerator/amp/pull/534)
+* add cli color theme [#495](https://github.com/appcelerator/amp/pull/495)
+* Fix stats issues [#404](https://github.com/appcelerator/amp/pull/404) [#403](https://github.com/appcelerator/amp/pull/403)
+
+### Vendoringttps://github.com/appcelerator/amp/pull/586
```
Comment (bquenin) [inline CHANGELOG.md]: Probably a typo here

**Review by bquenin — CHANGES_REQUESTED** [review]
Also, it seems that 0.3.0 CHANGELOG has disappeared

File: CHANGELOG.md (hunk 1 above)
Comment (JosephGJ) [reply CHANGELOG.md]: Removed.

**Review by bquenin — APPROVED** [review]

**Review by bquenin — DISMISSED** [review]
Also, it seems that 0.3.0 CHANGELOG has disappeared

**Comment by subfuzion** [comment]
LGTM. Thanks @ndegory 👍

**Pull request merged**